File: AzureQueues/sendMessage.rb
require 'waz-queues'

WAZ::Storage::Base.establish_connection!(:account_name => "prdc", 
			 :access_key =>  "6mEliBWPnykttQyZsLS9JJIY4xYNWMXLt3AFsH7FIRnkxLjAPkXQk2kvM/uiQnSidRUvY7DdA2OnW8AFRh0scA==", 
			 :type_of_service => "queue", 			 :base_url => "prdc.queue.core.windows.net")

queue = WAZ::Queues::Queue.ensure("prdc")

queue.enqueue!(Base64.encode64("ruby says hi"))

File: AzureQueues/sendMessage.py
from azure.storage import *

queue_service = QueueService(account_name='prdc', account_key="6mEliBWPnykttQyZsLS9JJIY4xYNWMXLt3AFsH7FIRnkxLjAPkXQk2kvM/uiQnSidRUvY7DdA2OnW8AFRh0scA==")

queue_service.create_queue('prdc')

queue_service.put_message('prdc', base64.b64encode('Hi from python'))